package.json

```diff
@@ -29,22 +29,24 @@
     "flow-bin": "^0.33.0",
     "lodash": "^4.15.0",
     "nyc": "^8.1.0",
+    "rollup": "^0.36.3",
+    "rollup-plugin-babel": "^2.6.1",
+    "rollup-plugin-node-resolve": "^2.0.0",
     "unicode-9.0.0": "~0.7.0"
   },
   "bin": {
     "babylon": "./bin/babylon.js"
   },
   "scripts": {
-    "build": "babel src --out-dir lib",
+    "build": "rollup -c",
     "coverage": "nyc report --reporter=json && codecov -f coverage/coverage-final.json",
     "lint": "eslint src bin",
     "flow": "flow",
     "prepublish": "cross-env BABEL_ENV=production npm run build",
     "preversion": "npm run test && npm run changelog",
     "test": "npm run lint && npm run flow && npm run build && npm run test-only",
     "test-only": "ava test",
-    "test-ci": "cross-env BABEL_ENV=test npm run build && nyc npm run test-only",
-    "watch": "babel src --out-dir lib --watch",
```

Comment:
I never used watch in babylon as the tests usually do a build before and the build does not take too long.